### DISCUSSION TITLE
Removal of Crop RNG tickets suggestion

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#3771] [Ideas]
I would heavily be for a change in removing tickets being discovered from harvesting crops.

What would this mean?

- Bert becomes more important (not necessarily a good thing, suggestion: remove Bert or make him a lot easier, not asking high-value items)
- 2 players doing the same chores/deliveries will be decided by xp. I foresee increased tie-breakers if we remove crop ticket rng. Not necessarily a bad thing, it would give more value to eating more food.

Should Bert be removed? Reworked?
The worst case scenario i foresee is that 100+ players have the same amount of points and will then all be decided by xp. Is this okay?

**Comment by brian-man** [comment]
`+` strategy
`-` RNG
👍

**Comment by Blasta-alasta** [comment]
XP being the decider was the reason we got the RNG tickets in the first place right...

I wonder if there are any ideas out there for alternative to both? Few thoughts...

- What if you had a second auction between tie breakers (i'm not sure, sounds like it could be hard to code/build though)? You could even have it for a random resource and the tiebreakers set their bids and highest wins. _There must be better ideas out there in the community than this surely lol._

- If you did switch to weekly chores, then it could be a case of whoever gets there first... but doesn't seem that fair on timezone sensitive players though.

- What if the tiebreaker was the only thing determined by RNG, Ie - only those that tied for the lowest bids, be it 2 or 20, the drum rolls and a winner is chosen?

- In the 'worst case scenario', this is only bad imo if the item in questions is super OP (say like Infernal Pitchfork)... otherwise anything else could be balanced out in the long run. But you still gotta have awesome items up for grabs, and you don't want everyone deciding "oh hey, i can just complete everything and get the best item" cos the supply will end up too high and players will be disappointed.

- Those being said... I kind of think if the RNG tickets do remain, that they should be rewarded through all farm activity, not just from crops. Like from mining, fishing, cooking... really reward those that are the most active farmers.

**Comment by Dionis404** [comment]
Crop RNG is a good mechanic that needs tweaking. 
It gives us a sense of excitement in the race. 
And allows you to pick up items for those players who do not have a very high level. 
By removing Crop RNG all auctions will be taken by players who have whale farms and are in the top 100 in experience. 
It is almost impossible to catch up with such players in terms of experience. 

We need a complex formula for calculating the drop chance, which will be hidden from the players that would not be a problem with the abuse of mechanics. 
Like now people who remove the scarecrow and lunar calendar get more season tickets.
